app/models/immunisation_import.rb

```diff
@@ -32,6 +32,7 @@
 #
 class ImmunisationImport < ApplicationRecord
   include CSVImportable
+  include VaccinationMailerConcern
```

Comment:
It doesn't seem right to include a controller concern in to a model. If we need to share this logic between models and controllers then a concern might not be the right place for it and instead we could have a service class.